embedchain/llm/huggingface.py

```diff
@@ -42,6 +43,7 @@ def _get_answer(prompt: str, config: BaseLlmConfig) -> str:
         else:
             raise ValueError("`top_p` must be > 0.0 and < 1.0")
 
+        logging.info(f"Using HuggingFaceHub with model {config.model}")
         llm = HuggingFaceHub(
             huggingfacehub_api_token=os.environ["HUGGINGFACE_ACCESS_TOKEN"],
             repo_id=config.model or "google/flan-t5-xxl",
```

Comment:
Not related to this PR but we should not do the `config.model or "google/flan-t5-xxl"` here. 

It should be before the logging statement that you added so that we capture the right model name in logging statement.

Reply:
let me quickly update that